From a121701610c176e3dc04a3740e789c2c9e3c8a8f Mon Sep 17 00:00:00 2001
From: Gwendal Grignou <gwendal@chromium.org>
Date: Mon, 8 Jan 2018 11:49:02 -0800
Subject: [PATCH] CHROMIUM: cros-ec: Update cros_ec_commands header to latest
 version

BUG=b:67743747
TEST=`emerge-cyan chromeos-kernel-4_14`

Change-Id: Icb007e0a591021d4ada3b7db73e7743c6cfde525
Signed-off-by: Gwendal Grignou <gwendal@chromium.org>

Conflicts:
	include/linux/mfd/cros_ec_commands.h

[rebase419(groeck): Context conflicts; improved alignment with chromeos-4.14]
Signed-off-by: Guenter Roeck <groeck@chromium.org>

[rebase419(rrangel): Most everything was already applied.]
Signed-off-by: Raul E Rangel <rrangel@chromium.org>
---
 include/linux/mfd/cros_ec_commands.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/include/linux/mfd/cros_ec_commands.h b/include/linux/mfd/cros_ec_commands.h
index 03250fbe6539..bfb063617656 100644
--- a/include/linux/mfd/cros_ec_commands.h
+++ b/include/linux/mfd/cros_ec_commands.h
@@ -1257,6 +1257,14 @@ enum ec_feature_code {
 	EC_FEATURE_FINGERPRINT = 28,
 	/* The MCU exposes a Touchpad */
 	EC_FEATURE_TOUCHPAD = 29,
+	/* The MCU has RWSIG task enabled */
+	EC_FEATURE_RWSIG = 30,
+	/* EC has device events support */
+	EC_FEATURE_DEVICE_EVENT = 31,
+	/* EC supports the unified wake masks for LPC/eSPI systems */
+	EC_FEATURE_UNIFIED_WAKE_MASKS = 32,
+	/* EC supports 64-bit host events */
+	EC_FEATURE_HOST_EVENT64 = 33,
 	/* EC supports CEC commands */
 	EC_FEATURE_CEC = 35,
 	/*
-- 
2.23.0.rc1.153.gdeed80330f-goog

